parser.c

```diff
@@ -88,7 +88,6 @@ static int parse_open_string(struct tinyiiod *iiod, char *str)
 {
 	char *device, *ptr;
 	long samples_count;
-	unsigned int i;
```

Comment:
There are some misspellings in the commit message: loock, iioosciloscope

Reply:
Signed-off is missing

Reply:
I think libiio is the one sending the commands iio-scope is just the application using libiio. This is just a guess i could be wrong.

Reply:
Fixed problems here